tc/sgx/common/crypto/sig_private_key.cpp

```diff
@@ -206,16 +206,13 @@ std::string pcrypto::sig::PrivateKey::Serialize() const {
 }  // pcrypto::sig::PrivateKey::Serialize
 
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
-// Computes SHA256 hash of message.data(), signs with ECDSA privkey and
-// returns ByteArray containing raw binary data
+// Signs message.data() with ECDSA privkey and returns ByteArray
+// containing raw binary data
```

Comment:
Can we change raw binary data -> signature bytes ?

Reply:
It loks fine to me. Signs and returns ByteArray containing raw binary data

Reply:
ok